tests/acceptance/features/apiNotification/notificationSetting.feature

```diff
@@ -0,0 +1,107 @@
+@email
+Feature: Notification Settings
+  As a user
+  I want to manage my notification settings
+  So that I do not get notified of unimportant events
+
+
+  Scenario: disable mail notification
```

Comment:
instead of checking 2 things in the same test scenario can we split it in two separate?
1. should receive notification for enabled mail notification (which is default i guess)
2. should not receive notification for disabled mail notification
its your call though.